src/IntegrationUtils.ts

```diff
@@ -509,4 +550,33 @@ export function killChildProcessForIntegration(integration:string) : Promise<boo
 		}
 		resolve(childProcess ? childProcess.killed : true);
 	});	
-}
\ No newline at end of file
+}
+
+function findChoiceFromStartsWith(inChoice: string | undefined) : string | undefined {
+	if (inChoice) {
+		if (devModeIntegration.startsWith(inChoice)) {
+			return devModeIntegration;
+		}
+		if (basicIntegration.startsWith(inChoice)) {
+			return basicIntegration;
+		}
+	}
+	// other choices not supported at present because they will require additional inputs
+	return undefined;
```

Comment:
Because quite honestly that is beyond the scope of this extension's code. There will always be some give and take between Didact and other extensions. Didact can call anything after all and there's no way to protect against that. We may very well add additional support to this code in the future for the other choices, but for now I think this is enough. We can perhaps add some documentation around what is supported for Didact calls, but codifying it is an unneeded step.

Reply:
Given all the complexity added to support the Didact case, it seems important to try to clarify all the things that we can. Fine to handle it in another iteration.

reported https://issues.redhat.com/browse/FUSETOOLS2-471